cmd/experimental/kueue-populator/test/verify-helm-install.sh

```diff
@@ -57,6 +57,7 @@ echo "Installing kueue-populator with Topology and ResourceFlavor..."
 "$HELM" upgrade --install kueue-populator charts/kueue-populator \
   --namespace kueue-system \
   --create-namespace \
+  --set kueue.enabled=true  \
```

Comment:
It’s really weird that we’re installing Kueue with the previous version.
I see we only use it in tests. I propose installing Kueue separately instead of using these values.

Reply:
> Later we could replace that with the real registry for release using the release script.

I think we should merge `kueue` and `kueue-populator` together. Like we did for kueueviz.

Reply:
After we move it to cmd directory we can add just a value on kueue charts. Like we did for kueueviz. Not now.